backend/cloud.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package backend
+
+type CloudData interface {
+	Copy() CloudData
+	Equals(CloudData) bool
+}
+
+type UnknownCloudData struct{}
+
+func (*UnknownCloudData) Copy() CloudData {
+	return &UnknownCloudData{}
+}
+
+func (*UnknownCloudData) Equals(d CloudData) bool {
+	_, ok := d.(*UnknownCloudData)
+	return ok
+}
+
+type Cloud struct {
+	Organization string
```

Comment:
This appears to be unused and we shouldn't need to be decoding the org anyway, or is there a particular use case for it?